modules/Bio/EnsEMBL/Utils/Tree/Interval/Mutable.pm

```diff
@@ -0,0 +1,129 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+  Please email comments or questions to the public Ensembl
+  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+  Questions may also be sent to the Ensembl help desk at
+  <http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+=head1 NAME
+
+Bio::EnsEMBL::Utils::Tree::Interval::Mutable
+
+=head1 SYNOPSIS
+
+
+=head1 DESCRIPTION
+
+Class representing a dynamic, i.e. mutable, interval tree implemented as an augmented AVL balanced binary tree.
+
+This module is a wrapper around two possible implementations: one using the Perl extension (XS) mechanisms, and
+a pure Perl (PP) one. 
+
+The module is capable of detecting whether the XS module is available and it loads it in that
+case; it falls back to the PP implementation otherwise.
+ 
+=head1 METHODS
+
+=cut
+
+package Bio::EnsEMBL::Utils::Tree::Interval::Mutable;
+
+use strict;
+
+use Data::Dumper;
```

Comment:
This module isn't used anywhere in this file, is it?

Reply:
(still here)

Reply:
Fixed.